core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt

```diff
@@ -81,5 +81,17 @@ public enum class DroidKaigi2023Day(
         public fun defaultDyamicThemeDate(): Boolean {
             return Day1.start < Clock.System.now()
         }
+
+        /**
+         * @return appropriate day at this time
+         */
+        fun currentDay(): DroidKaigi2023Day {
+            lateinit var currentDay: DroidKaigi2023Day
+            for (entry in entries.sortedBy { it.day }) {
+                currentDay = entry
+                if (Clock.System.now() <= entry.end) break
```

Comment:
We would like to display Day1 tab after the conference ended.
Would you modify the logic? 🙏

Reply:
Thank you for comment. I modified the logic to select Day1 when it is after the conference ended.